src/main/java/com/smalaca/messagesender/controller/UserController.java

```diff
@@ -9,26 +11,37 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
+@RequestMapping("/user")
 @RestController
 public class UserController {
 
     @Autowired
     private UserCrud userCrud;
 
-    @RequestMapping("/user/create")
-    public String createUser(@ModelAttribute UserDto userDto) {
-        if (userCrud.createUser(userDto))
-            return "User Created";
-        else
-            return "User Creation Failed!";
+    @RequestMapping("/create")
+    public Response createUser(@ModelAttribute UserDto userDto) {
+        return userCrud.createUser(userDto);
+    }
+
+    @RequestMapping("/block")
+    public Response blockUser(@RequestParam String login) {
+        return userCrud.blockUser(login);
     }
 
-    @RequestMapping("/user/block")
-    public String blockUser(@RequestParam String login) {
-        if (userCrud.blockUser(login)) {
-            return "User Blocked";
-        }
-        return "User Block Failed!";
+    @RequestMapping("/show/all")
+    public List<User> showAllUsers() {
+        return userCrud.showAllUsers();
     }
 
+    @RequestMapping("/update")
+    public Response updateUser(@ModelAttribute UserDto userDto) {
+        return userCrud.updateUser(userDto);
+    }
+
+    @RequestMapping("/show")
+    public User showInfoAboutUser(@RequestParam String login) {
```

Comment:
po "REST'owemu" lepiej byłoby mieć:
```java
@RequestMapping("/show")
public List<User> showAllUsers() { /** ... */ }

@RequestMapping("/show/{id}")
public List<User> showUser(@PathVariable String id) { /** ... */ }
```
ale to tak na przyszłość, albo jeżeli bardzo Ci się nudzić będzie 😄